### ISSUE TITLE
Need help on: 

### ISSUE THREAD

**Issue by boxtesign** [diegoazh/gmap-vue#99]
# What you try to do
**Describe the bug**
autocomplete with custom text field doesn't return places data.

**To Reproduce**
```
<gmap-autocomplete>
                      <template v-slot:input="slotProps">
                        <v-text-field
                          ref="input"
                          outlined
                          placeholder           
                          label="Address"
                          color="teal"
                          @place_changed="setPlace"
                          v-on:listeners="slotProps.listeners"          
                        ></v-text-field>
                      </template>
</gmap-autocomplete>
```
```
setPlace(place) {
      console.log("test")
      this.place = place;
      console.log(this.place);
    },
```
**Expected behavior**
Print places data in console. Here is the that works:

```
<gmap-autocomplete
    placeholder="This is a placeholder text"
    @place_changed="setPlace"
 ></gmap-autocomplete>
```
```
setPlace(place) {
      console.log("test")
      this.place = place;
      console.log(this.place);
    },
```

**Current behavior**
The code below with custom text field does't work. I copied the code from the documentation.

**Screenshots**
none

**Desktop (please complete the following information):**
 - OS: [Mac]
 - Browser [chrome, safari]

**Smartphone (please complete the following information):**

**Additional context**

## Versions

- NPM:

## Package manager

- [ ] NPM

## Plugin version

- [ ] 1.4.0

Explain to us what you are trying to do or what is your goal.

# What is the problem

Explain to us what is the problem or what is the part of the documentation that is not clear.

# What kind of things have you already treated

A small description of the solutions that you already treated.

# Steps to reproduce

Explain to us how to get the same problem or provide a small example repository to work on it.

## Os

- [ ] Mac Os X
- [ ] Linux (write here the name and version of your distribution)
- [ ] Windows

## Mobile

- [ ] iOS
- [ ] Android

## Versions

- Node:
- NPM:
- Yarn:

## Package manager

- [ ] NPM
- [ ] Yarn

## Plugin version

- [ ] 1.4.0
- [ ] 1.2.2
- [ ] 1.2.1
- [ ] 1.2.0
- [ ] 1.0.0

**Comment by diegoazh** [comment]
@boxtesign please test the new version 2.0.0 and let me know if the issue persists.

**Comment by diegoazh** [comment]
duplicate #98
solved on #98
